python/triton/_C/include

```diff
@@ -1 +1 @@
-../../../include/
\ No newline at end of file
+../../../include/
```

Comment:
For some reason, on Windows `pre-commit` adds a new-line character. This change seems quite correct to me. Let me know if I need to revert it.

Reply:
It seems to break the CI, so I reverted the changes.